src/libs/convertToLTR/index.ts

```diff
@@ -1,5 +1,13 @@
+import CONST from '../../CONST';
 import ConvertToLTR from './types';
 
-const convertToLTR: ConvertToLTR = (text) => text;
+const convertToLTR: ConvertToLTR = (text) => {
```

Comment:
Since we also have `index.android.ts` file for android which works similar, I wonder if we still need it?

Reply:
@burczu, wherever we utilize the convertToLTR function, it appears to reference this file. I believe we can safely eliminate ` index.android.ts `. Should I proceed with removing it and pushing the code?

Reply:
Well, that's how it works - you are referencing the `index.ts` file and RN uses `index.android.ts` file under the hood for Android devices, so after removing this file it needs to be tested on Android if it doesn't brake anything.

Reply:
Sure I will test it in a min and will get back to you

Reply:
@burczu I have tested it on android and its not breaking anything.